src/services/kpis/kpis.svc.coffee

```diff
@@ -175,6 +175,12 @@ angular
       templ = _self.getKpiTemplate(kpiEndpoint, kpiWatchable)
       ((templ? && templ.target_placeholders?) && templ.target_placeholders[kpiWatchable]) || {}
 
+    @getApiV2KpiDataKey = (kpi)->
+      # Formats the kpi endpoint to select the key name
+      # e.g response.cash_projection = { triggered: true, ... }
+      # TODO: maybe the 'kpi' of endpoint 'kpis/cash_projection' should be removed?
```

Comment:
Yes: I don't think the endpoint was returning `kpis/something` in the API v1?
Can you please align the Bolt endpoint?